build/source/netcdf/modelwrite.f90

```diff
@@ -274,7 +274,7 @@ subroutine writeData(modelTimestep,outputTimestep,nHRUrun,maxLayers,meta,stat,da
       case(iLookVarType%ifcSoil); maxLength = nSoil+1
       case default; cycle
      end select ! vartype
-
+     
```

Comment:
Please set editor preferences to remove trailing white space

Reply:
@martynpclark Set. Also, removed from these files.